core/server/api/v2/index.js

```diff
@@ -103,10 +103,6 @@ module.exports = {
         return shared.pipeline(require('./slack'), localUtils);
```

Comment:
Every content API ctrl uses the pattern `{resource}-public`, except of this ctrl. Consistency change.